### PR TITLE
docs: update installation steps

### LINKED ISSUE #97: TRG 1.02: Improve INSTALL.md
### Description

https://github.com/eclipse-tractusx/sd-factory/blob/v2.1.9/INSTALL.md

IMO there's quite a bit of confusing information in there, for instance:

- `1.) Using helm commands:` and `2.) Local installation:` should maybe rather be called `Installation from released chart` and `Installation from repository`
- with the command in 1.a your not `adding helm repository in tractusx` but you add the chart-repository from tractusx and to add that with the name `sd-factory` is quite misleading
- 2.c and 2.d are contradictory
- 2.e didn't try it myself but are you sure that that step even work as described

It's not a blocking issue for the QG check for 24.3 release but please make the change for the next release.

### PR DESCRIPTION
**Pull request by adkumar1**
- Closes #97 
- Updated installation steps in INSTALL.md file

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
 
-1.) Using helm commands: <br />
+1.) Installation from released chart: <br />
 
 How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
-    a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    a.) Add helm repository for sdfactory:
```
Comment (evegufy) [inline INSTALL.md:10]: It's not the helm repository for sd factory, it's helm chart repository for tractusx-dev

File: INSTALL.md [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
+           helm repo add [name] https://eclipse-tractusx.github.io/charts/dev
```
Comment (evegufy) [inline INSTALL.md:11]: would be more understandable if you'd use tractusx-dev instead of [name]

File: INSTALL.md [hunk 3]
```diff
@@ -2,24 +2,25 @@
...
     b.) To search the specific repo in helm repositories 
-           helm search repo sd-factory/sdfactory
+           helm search repo [name]/sdfactory
     c.) To install using helm command:
-           helm install sdf sd-factory/sdfactory
+           helm install [ReleaseName] [name]/sdfactory
```
Comment (evegufy) [inline INSTALL.md:15]: same as in the 2.e, this command installs on a k8s cluster; maybe mention that.
also, in 2.e you mention a namespace, not sure why here not as well.

File: INSTALL.md [hunk 4]
```diff
@@ -2,24 +2,25 @@
...
 
-2.) Local installation:
+2.) Installation from repository:
 
     a.) git clone https://github.com/eclipse-tractusx/sd-factory.git
-    b.) Modify values file according to your requirement
-    c.) You need to define the secrets as well in values.yaml
+    b.) cd sd-factory
+    c.) Modify values file according to your requirement
+    d.) You need to define the secrets as well in values.yaml
```
Comment (evegufy) [inline INSTALL.md:23]: Not sure why there isn't an analog point under 1, especially in the case of installing from the released chart it should be described how to manage configuration, for instance by using Hashicorp vault, that you mentioned earlier here under 2.

File: INSTALL.md [hunk 5]
```diff
@@ -2,24 +2,25 @@
 
 Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
```
Comment (evegufy) [inline INSTALL.md:3]: There is just one helm chart, so using plural is confusing and maybe refer to the charts directory and the README there as well.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 1 above)
Comment (adkumar1) [reply INSTALL.md]: Taken care

File: INSTALL.md (hunk 3 above)
Comment (adkumar1) [reply INSTALL.md]: This has been taken care

File: INSTALL.md [hunk 6]
```diff
@@ -34,7 +35,6 @@ How to install application using helm:  <br />
               clearingHouseClientId: -> Client id for Clearing House
               clearingHouseClientSecret: -> Clearing house for Client secret
 
-    d.) These secrets should be defined in Hashicorp vault
     e.) Deploy in a kubernetes cluster
-        helm install sdfactory charts/SDFactory/ -n NameSpace
+        helm install [name] charts/sdfactory/ -n [NameSpace]
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply INSTALL.md]: This has been taken care

File: INSTALL.md (hunk 5 above)
Comment (adkumar1) [reply INSTALL.md]: This has been taken care.

File: INSTALL.md [hunk 7]
```diff
@@ -2,24 +2,25 @@
...
 
 
-2.) Local installation:
+2.) Installation from repository:
 
     a.) git clone https://github.com/eclipse-tractusx/sd-factory.git
-    b.) Modify values file according to your requirement
-    c.) You need to define the secrets as well in values.yaml
+    b.) cd sd-factory
+    c.) Modify values file according to your requirement
```
[the remark this answers is not in the document]
Comment (adkumar1) [reply INSTALL.md:23]: This has been taken care

File: INSTALL.md (hunk 4 above)
Comment (adkumar1) [reply INSTALL.md]: This has been taken care

File: INSTALL.md [hunk 8]
```diff
@@ -1,25 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+Helm chart is provided inside https://github.com/eclipse-tractusx/sd-factory
```
Comment (evegufy) [inline INSTALL.md:3]: Please make the sentence nicer:
A helm chart is provided inside the [charts](./charts) directory.

File: INSTALL.md [hunk 9]
```diff
@@ -1,25 +1,27 @@
...
-2.) Local installation:
-
+2.) Installation from repository:
+        
     a.) git clone https://github.com/eclipse-tractusx/sd-factory.git
-    b.) Modify values file according to your requirement
-    c.) You need to define the secrets as well in values.yaml
+    b.) cd sd-factory
+    c.) Modify values file according to your requirement
+    d.) You need to define the secrets as well in your-values.yaml. These secrets should be defined in Hashicorp vault to keep them secure
```
Comment (evegufy) [inline INSTALL.md:24]: The whole part regarding "These secrets should be defined in Hashicorp vault ..." would be more appropriate after 1.c. Then just remove 2.c and 2.d. (they are redundant) and change 2.e to 2.c.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 8 above)
Comment (adkumar1) [reply INSTALL.md]: Updated with appropriate sentence

File: INSTALL.md (hunk 9 above)
Comment (adkumar1) [reply INSTALL.md]: Modified the statements

File: INSTALL.md [hunk 10]
```diff
@@ -1,40 +1,27 @@
...
+    a.) Add helm repository for tractusx-dev:
+           helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
     b.) To search the specific repo in helm repositories 
-           helm search repo sd-factory/sdfactory
-    c.) To install using helm command:
-           helm install sdf sd-factory/sdfactory
+           helm search repo tractusx-dev/sdfactory
+    c.) To set your own configuration and secret values, install the helm chart with your own values file in kubernetes cluster  
+           helm install -f your-values.yaml [ReleaseName] tractusx-dev/sdfactory -n [NameSpace]
+    d.) These secrets should be defined in Hashicorp vault to keep them secure.
```
Comment (evegufy) [inline INSTALL.md:17]: I find a separate point for it confusing as this statement is just a clarification to 1.c, I'd just put it after 1.c.
Also, [this](https://github.com/eclipse-tractusx/sd-factory/pull/104/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0L23) list could be helpful, please re-add it

File: INSTALL.md [hunk 11]
```diff
@@ -1,40 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+A helm chart is provided inside the [charts](https://github.com/eclipse-tractusx/sd-factory/tree/main/charts/sdfactory) directory
```
Comment (evegufy) [inline INSTALL.md:3]: I recommend using a relative link and not linking directly to the main branch: [charts](./charts)

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by evegufy — CHANGES_REQUESTED** [review]
I also noticed that https://github.com/eclipse-tractusx/sd-factory/blob/main/charts/sdfactory/README.md is out of date due to the change from https://github.com/eclipse-tractusx/sd-factory/issues/93 and https://github.com/eclipse-tractusx/sd-factory/issues/91
Please update it and please also reference the README in the charts directory in the install.md here

File: INSTALL.md (hunk 11 above)
Comment (adkumar1) [reply INSTALL.md:3]: Should I mention like this ?
A helm chart is provided inside the charts directory in https://github.com/eclipse-tractusx/sd-factory repo

File: INSTALL.md (hunk 10 above)
Comment (adkumar1) [reply INSTALL.md]: This has been taken care

File: INSTALL.md (hunk 11 above)
Comment (adkumar1) [reply INSTALL.md]: Updated with relative link

**Comment by adkumar1** [comment]
> I also noticed that https://github.com/eclipse-tractusx/sd-factory/blob/main/charts/sdfactory/README.md is out of date due to the change from #93 and #91 Please update it and please also reference the README in the charts directory in the install.md here

Updated this as well.

**Review by evegufy — APPROVED** [review]

**Pull request merged**